data/en/mbs_0111.json

```diff
@@ -1,321 +1,380 @@
-                            {
-                                "type": "Date",
-                                "id": "reporting-period-answer-3-1",
-                                "label": "To",
-                                "mandatory": true,
-                                "q_code": "12"
-                            }
-                        ],
-                        "description": ""
-                    }]
-                },
-                {
-                    "type": "Question",
-                    "id": "turnover-block-4",
-                    "title": "Turnover",
-                    "questions": [{
-                        "type": "General",
-                        "guidance": {
+                        "preview_content": {
+                            "id": "preview",
+                            "title": "Information you need",
                             "content": [{
-                                    "list": [
-                                        "Commission",
-                                        "Costs incurred and passed on to customers",
-                                        "Sales of goods purchased for resale",
-                                        "Payment for work in progress",
-                                        "Income from sub-contracted activities",
-                                        "Revenue earned from other parts of the business not named  (please supply at fair value)"
-                                    ],
-                                    "title": "Include:",
-                                    "description": ""
+                                "description": "You can select the dates of the period you are reporting for, if the given dates are not appropriate."
+                            }],
+                            "questions": [{
+                                    "question": "Total Turnover",
+                                    "content": [{
+                                            "description": "Include:",
+                                            "list": [
+                                                "exports",
+                                                "payments for work in progress",
+                                                "costs incurred and passed on to customers",
+                                                "income from sub-contracted activities",
+                                                "commission",
+                                                "sales of goods purchased for resale",
+                                                "revenue earned from other parts of the business not named (please supply at fair value)"
+                                            ]
+                                        },
+                                        {
+                                            "description": "Exclude:",
+                                            "list": [
```

Comment:
This list does not match the order of the slides i.e. 

Exclude:
VAT
income from the sale of fixed capital assets
grants and subsidies
insurance claims
interest received